lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -42,19 +33,42 @@ def call(package)
 
         private
 
+        def boolean_error(manifest)
+          {
+            requirements_only: 'requirementsOnly',
```

Comment:
Can we get this from `RUBY_TO_JSON` rather than duplicating the JSON key? I know it won't actually change, but you've removed the duplication so well otherwise :)